system_tests/vision.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""System tests for Vision API."""
+
+import os
+import unittest
+
+from google.cloud import exceptions
+from google.cloud import storage
+from google.cloud import vision
+
+from system_test_utils import unique_resource_id
+from retry import RetryErrors
+
+
+class Config(object):
+    CLIENT = None
+    TEST_BUCKET = None
+
+    LOGO_FILE = os.path.join(os.path.dirname(__file__), 'data', 'logo.png')
```

Comment:
This doesn't need to be in `Config`, the `Config` is intended for run-time only values, but this one can be determined at import time without issue. It helps to include the docstring on `Config` to clarify that goal.

Also, filesystem paths can be annoying sometimes. I recommend the following (as globals, not as attrs on `Config`):

```python
_SYS_TESTS_DIR = os.path.abspath(os.path.dirname(__file__))
LOGO_FILE = os.path.join(_SYS_TESTS_DIR, 'data', 'logo.png')
```